### ISSUE TITLE
Camera is not responding

### ISSUE THREAD

**Issue by adjudaris** [jordenc/com.synology.surveillance#8]
Homey version 2.04
Synology version 2.0.0

Just updated the app, Camera is not responding anymore.
When movemend is detected by philio motion sensor, start recording or
When doorbel is pushed take photo

When testing the flows, get a green sign immidately when starting recording
when testing the take photo the card is keep spinning whit no result. 

Deinstalled the app, installed it new.
Nas is detected also the camera's and I can make new flows but with the same results.

**Comment by jordenc** [comment]
Sadly, there currently is no easy way to detect errors. Are you experienced with installing apps from the commandline?

**Comment by adjudaris** [comment]
Hello Jordan, 

Do you mean through cli? 
I have done that before. 

⁣BlueMail voor Android downloaden ​

Op 22 feb. 2019 13:54, om 13:54, Jorden <notifications@github.com> schreef:

**Comment by jordenc** [comment]
Yes exactly. (CLI = command line interface)

If you can, please install the Synology app through CLI and see if you get any errors in the logging when actions go wrong. Post some of them here (make sure it doesn't include your IP / username / password, just to be safe)

**Comment by adjudaris** [comment]
Oke, will do/try that later to day and let you know. 

⁣BlueMail voor Android downloaden ​

Op 22 feb. 2019 14:45, om 14:45, Jorden <notifications@github.com> schreef:

**Comment by adjudaris** [comment]
Installed app through CLI, same results thus none.
Where can I find the error loging?

I installed the camera's first with a new account as mentioned in your comments, after that I gave the account read/write in the NAS.
Then I tryed this again with my own account.

**Comment by adjudaris** [comment]
After installing the app through CLI, I been looking into Synology and Surveillance station but found nothing that has changed after version 1.2.9

But is far my knowledge goes, there is one strange thing!
Stopping Surveillance station then:
Removed one camera from homey tried to reconnect without succes.
Tried to start the flow where the camera needs to get starting, strange that within seconds I had the
green mark.
For some reason it seems that the app is not connecting to surveillance station, while in the logs before yesterday the camera was started and stopped on the welknown time.

Hopefull you can do something with this because with your app and a motion sensor spiders could move in front of the camera lens as much as the want.

**Comment by adjudaris** [comment]
In my case problem solved for one camera.
I have two camera's different IP and different port on my router.
One camera is'nt active most of the time and is powerd off the second one is outdoors and always active.
This morning I activated the first camera and to my supprise, when testing a flow with the second camera, the first one started recording. I removed the first camera from homey en the second one started recording.
After that removed the app and installed the app from the app store and added only the second camera. My flows for the second camera are working now but I can't install both cameras. When the first one is also installed the flows are always starting this camera.
